woocommerce/woocommerce
# Monorepo: update git postmerge hook.

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Update git postmerge hook to rely on lockfile instead of resove/install/update lock cycle..

### How to test the changes in this Pull Request:

- Perform local merge (e.g. a local outdated branch, merge trunk into it) - `pnpm install` should run about twice faster and install the needed deps just relying on lock file.
